src/mac.rs

```diff
@@ -672,12 +903,175 @@ pub enum DecodeError {
     /// The frame has the security bit set, which is not supported
     SecurityNotSupported,
 
-    /// The frame compresses the PAN ID, which is not supported
-    PanIdCompressNotSupported,
+    /// The frame's address mode is invalid
+    InvalidAddressMode(u8),
 
     /// The frame's address mode is not supported
-    AddressModeNotSupported(u8),
+    AddressModeNotSupported(AddressMode),
```

Comment:
Is this variant used anywhere? I can't find a use, but it seems there hasn't been a warning on CI. I'm confused :-)

Reply:
No, i cannot find the use of this error. I will remove the error.

Reply:
I will remove the unused error.